frontend/src/libs/components/meditation-tracks-list/meditation-track/styles.module.scss

```diff
@@ -0,0 +1,81 @@
+.track {
+  position: relative;
+  width: 100%;
+  max-width: 451px;
+  height: 177px;
+  margin: 0 auto;
+  border-radius: 30px;
+  box-shadow: 0 4px 23px 0 var(--violet-100);
+}
+
+.background-image {
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: block;
+  width: 37%;
+  height: 100%;
+  border-radius: 30px 0 0 30px;
+}
+
+.content {
+  position: absolute;
```

Comment:
it's better to avoid position absolute and in this case it's not needed, layout can be made without it

Reply:
fixed